apps/new/widget/page/project/QuickView.jsx

```diff
@@ -80,12 +80,13 @@ const BackgroundImage = styled.div`
 const id = `${project.accountId}/project/${project.projectID}`;
 
 return (
-  <Container>
+  <Container style={{ opacity: 1 }}>
```

Comment:
Why not add this to the styled component rather than style attribute

Reply:
Sure sure